Lib/test/test_zipimport.py

```diff
@@ -41,7 +41,8 @@ def make_pyc(co, mtime, size):
         else:
             mtime = int(-0x100000000 + int(mtime))
     pyc = (importlib.util.MAGIC_NUMBER +
-        struct.pack("<iii", 0, int(mtime), size & 0xFFFFFFFF) + data)
+        struct.pack("<iLL", 0,
```

Comment:
As I noticed in the previous PR, the lines above can be removed.

Reply:
Done, gonna wait for the Mac CI to run on this.